test/extended/util/framework.go

```diff
@@ -61,12 +61,35 @@ func WaitForOpenShiftNamespaceImageStreams(oc *CLI) error {
 				e2e.Logf("ImageStream Error: %#v \n", err)
 				return false
 			}
-			for tag := range is.Spec.Tags {
+			for i, tag := range is.Spec.Tags {
 				e2e.Logf("Checking tag %v \n", tag)
```

Comment:
i think you want this to print `i` not `tag`.  You want to print the key/tagname, not the actual tag content, I would assume.

Reply:
(if your goal was to have more diagnostics here, i'd suggest only printing the full tag object in the error cases)